SU-HSDO/suhumsci
# Colorful theme Events views

# READY FOR REVIEW

## Summary
This adds the Date-stacked vertical card, and styles it to look like [the designs](https://www.figma.com/file/toNZnFLadtpNpG5dPzNGWz/%E2%9A%99%EF%B8%8F-Colorful-Theme---Working?node-id=2152%3A45).

![image](https://user-images.githubusercontent.com/1256329/73718950-e915cb80-46eb-11ea-8671-c69e8de9e097.png)

It also adds support for colorful bolded field labels to existing "card" patterns (horizontal, vertical, date-stacked):

<img width="649" alt="Screen Shot 2020-02-04 at 1 17 52 AM" src="https://user-images.githubusercontent.com/1256329/73719075-427dfa80-46ec-11ea-9239-f9ab76f6b3f1.png">

For more details see: https://sparkbox.atlassian.net/browse/STN-89

## Need Review By (Date)
'asap'

## Urgency
high

## Steps to Test
1. Create a clone of the hs_events view.
2. Customize the display to use the "Date-stacked Vertical Card," and populate all it's fields with corresponding data from the event.
3. Place the view on the page and confirm that it matches the designs.

(I'll see if I can get a Views config dump for this, but I may need to set it up.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
